portality/tasks/harvester_helpers/workflow.py

```diff
@@ -1,34 +1,56 @@
+from portality.api.v2 import ArticlesCrudApi
 from portality.api.v2.client import client as doajclient
 from portality.core import app
 from portality.lib import plugin
-from portality.decorators import capture_sigterm
+from portality.models import Journal, Account
 from portality.models.harvester import HarvesterProgressReport as Report
 from portality.models.harvester import HarvestState
 from portality.lib.dataobj import DataObjException
+from portality.settings import BASE_FILE_PATH
 
 
 class HarvesterWorkflow(object):
+    logger = None
```

Comment:
It doesn't feel right to create a per-run logger on a global singleton member variable (it may have side-effects which could cause runtime errors down the line).  We should either:

1. Pass the logger in wherever it is needed by a `@classmethod`
2. Change the HarvesterWorkflow class so that it is instantiated when it is run, with a logger as an instance member variable